package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@energyweb/iam-roles",
+  "name": "ew-credentials",
```

IMO, it still makes sense to have the packages namespaced under `@energyweb`. What do you think @nichonien ?
```suggestion
  "name": "@energyweb/ew-credentials",
```

---

@jrhender I thought of it but then `@energyweb/ew-credential-governance` is a long package name. Also, `ew` does resemble `energyweb` in a shorter way/form.

---

Hmm, IMO, all of energyweb's packages should be scoped under `@energyweb`. I can see how it seems a bit redundant but one could argue that `@energyweb` refers to the organization publishing the package and `ew` refers to the fact that we are doing credential governance in our more specific way.
If I had to chose between `@energyweb` and `ew`, I would chose `@energyweb`. So maybe the package name should be `@energyweb/credentials`. @mhrsntrk @jcmelati what are your thoughts here?

---

Ok, after discussing with @mhrsntrk in standup, we will proceed with `@energyweb/credentials`